### PR TITLE
feat: Allow replicas to be set for application-controller

### PR DESCRIPTION
**Pull request by mr-sour**
Checklist:

* [X] I have update the chart version in `Chart.yaml` following Semantic Versioning.
* [X] Any new values are backwards compatible and/or have sensible default.
* [X] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed the CLA and the build is green.
* [x] I will test my changes again once merged to master and published.

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

**Comment by mr-sour** [comment]
Fixes https://github.com/argoproj/argo-helm/issues/492

**Comment by mr-sour** [comment]
I'll fix all those checks soon :tm:  also something odd is going when I was testing it worked on a fresh instance but updating my chart on a existing instance the changes didn't seem to to be available. I doubt its related to this change but a curiosity indeed.

**Comment by spencergilbert** [comment]
Thanks for the addition! Can you resolve the CI issues?

**Comment by mr-sour** [comment]
Yes I will resolve the issues as soon as I figure out why git hates me. Or maybe I need to _git_ better with git 😄

**Comment by mr-sour** [comment]
So based on my workflow of how I was trying to amend those commits I was actually orphaning commits outside of any branch where it would then make the DCO thing very upset. I have since fixed it all.

**Review by spencergilbert — APPROVED** [review]
Awesome, thank you! I forgot/didn't realize signing was required now

File: charts/argo-cd/values.yaml [hunk 1]
```diff
@@ -31,6 +31,10 @@ controller:
     tag: # v1.7.6
     imagePullPolicy: # IfNotPresent
 
+  # If changing the number of replicas you must pass the number as ARGOCD_CONTROLLER_REPLICAS as an environment variable
```
Comment (Junnplus) [inline charts/argo-cd/values.yaml:34]: why not auto set environment variable in template?

**Pull request merged**